src/Symfony/Component/Translation/Loader/XliffFileLoader.php

```diff
@@ -48,14 +48,14 @@ public function load($resource, $locale, $domain = 'messages')
         foreach ($xml->xpath('//xliff:trans-unit') as $translation) {
             $attributes = $translation->attributes();
 
-            if (!(isset($attributes['resname']) || isset($translation->source)) || !isset($translation->target)) {
+            if (!(isset($attributes['resname']) || isset($translation->source))) {
```

Comment:
Of course this could be changed. We simply left it as it is to keep the PR as small as possible.
Depends on the reader which one is more easy to read anyway. ;)